src/mono/wasm/runtime/invoke-js.ts

```diff
@@ -29,6 +29,70 @@ export function mono_wasm_bind_js_function(function_name: MonoStringRef, module_
         const fn = mono_wasm_lookup_function(js_function_name, js_module_name);
         const args_count = get_signature_argument_count(signature);
 
+        const arg_marshalers: Function[] = [];
```

Comment:
for these instead of `[]` you may do `new Array(args_count)`, it's not a big deal though since this is not a hot path. if you do that you need to fill the slots in via index `arr[index]` instead of via push.